src/Network/Network/ChangeLog.md

```diff
@@ -24,6 +24,7 @@
     - Added `TLSv1_3` to the validation set of `MinProtocolVersion`
     - Removed validation for null or empty cipher suites list since there can be empty cipher suites list for min protocol version of tls1.3
 * [Breaking Change] Changed default value of `-PrivateEndpointNetworkPoliciesFlag` to `Disabled` in `Add-AzVirtualNetworkSubnetConfig` and `New-AzVirtualNetworkSubnetConfig`
+* Onboard KubernetesConfiguration PrivateLink Scopws to Private Link Common Cmdlets
```

Comment:
Please move the change log message to the bottom of the title of "## Upcoming Release" to make sure all change logs are added into latest release.